proxy/secret.go

```diff
@@ -66,7 +66,7 @@ func GetSecretList(gateway string, tlsInsecure bool) ([]schema.Secret, error) {
 	return results, nil
 }
 
-// RemoveSecret remove a secret via the OpenFaaS API by name
+//RemoveSecret remove a secret via the OpenFaaS API by name
```

Comment:
What tool is removing the space between the comment and `//`? I use vscode and its built-in formatters that keep putting it back.